app/scripts/controllers/transactions/index.js

```diff
@@ -126,6 +126,11 @@ const VALID_UNAPPROVED_TRANSACTION_TYPES = [
  * @param {number} [opts.txHistoryLimit] - number *optional* for limiting how many transactions are in state
  * @param {Function} opts.hasCompletedOnboarding - Returns whether or not the user has completed the onboarding flow
  * @param {object} opts.preferencesStore
+ * @param {object} hooks
+ * @param {Function} hooks.afterSign - Additional logic to execute after signing a transaction. Return false to not change the status to signed.
+ * @param {Function} hooks.beforeApproveOnInit - Additional logic to execute logic before approving the transaction when initialized. Return false to prevent the transaction to approved.
+ * @param {Function} hooks.beforePublish - Additional logic to execute before publishing the transaction. Return false to prevent to broadcast of the transaction.
```

Comment:
```suggestion
 * @param {Function} hooks.beforePublish - Additional logic to execute before publishing a transaction. Return false to prevent the broadcast of the transaction.
```